3-fleetscope/modules/env_baseline/mcg.tf

```
@@ -30,7 +30,7 @@ resource "google_gke_hub_feature" "mci" {
 
   depends_on = [
     google_gke_hub_feature.mcs,
```

likely need to remove the comma

```suggestion
    google_gke_hub_feature.mcs
```